core/main_test.ts

```diff
@@ -21,6 +21,10 @@ defaultDataset: defaultDataset
 defaultLocation: US
 `;
 
+const VALID_WORKFLOW_SETTINGS_YAML_WITHOUT_PROJECT_DATASET_DEFAULTS = `
+defaultLocation: US
+`;
+
```

Comment:
Nit: inline this to the test, it's only used once and the name is more complex than the string

Reply:
Done.